.github/stale.yml

```diff
@@ -0,0 +1,69 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 60
+
+# Number of days of inactivity before an Issue or Pull Request with the stale label is closed.
+# Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
+daysUntilClose: 7
```

Comment:
This would be very harsh if someone is in the middle of things but goes on PTO for example.

Reply:
What do you suggest as alternative ? (7 days is the one I've seen used the most and the default)

Reply:
14 maybe?